app/assets/stylesheets/locations.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Locations controller here.
```

Comment:
We can delete this file.  We won’t need it

Reply:
file deleted